src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke.task;
+
+/**
+ * The class Task denotes a task.
+ *
+ * @author Alvin Chee
+ */
+public class Task {
+    private String taskInfo;
+    private TaskType taskType;
+    protected boolean done;
+    public static final int undoneNo = 0;
+    public static final int doneNo = 1;

```

Comment:
Maybe these name could be changed to something clearer like doneNum?

Reply:
Noted with thanks! :)